ui/pages/send/send-content/send-content.component.js

```diff
@@ -19,6 +20,7 @@ import SendGasRow from './send-gas-row';
 export default class SendContent extends Component {
   state = {
     showNicknamePopovers: false,
+    isContractAddress: false,
```

Comment:
I think for clarity we should call this `isKnownContractAddress`

Reply:
Fixed in: ad39793344c75085bf0a75435319383fa7f7b514